### DISCUSSION TITLE
This api[ `/api/v5/publish` ], when the status code is 400, the error message should not be an array.

### DISCUSSION THREAD

**Discussion by ArcherTrister** [emqx/emqx#11485] [Ideas]
![VKY@{J)BP}5E$)4@N0EN%BP](https://github.com/emqx/emqx/assets/35516482/b1b6c2d5-4643-4a33-90e8-d5dfa47a1f39)

**Comment by kjellwinblad** [comment]
I agree that this seems weird. Unfortunately it is probably too late to change it if this is really the case and not just a bug in documentation. I will convert this to an issue and we will investigate if this is just a documentation bug or something else.

**Comment by kjellwinblad** [comment]
Issue: https://github.com/emqx/emqx/issues/11488

We will use this issue to track this so I will close this discussion.
